site/en/models.md

```diff
@@ -114,6 +120,40 @@ This will download and store the resource in the [proper location](#where-is-the
 ocrd-tesserocr-recognize -P model mymodel
 ```
 
+### Models and Docker
+
+If you are using OCR-D with Docker, we recommend keeping all downloaded resources in a persistent host directory,
+separate of the `ocrd/*` Docker container and data directory, and mounting that
```

Comment:
```suggestion
separate of the OCR-D Docker container(s) and data directory, and mounting that
```

Once we switch to `ghcr.io` the prefix will be `OCR-D` not `ocrd`. Best to not be too specific.